app/events/db/migrations/db_table_sql_scripts.sql

```diff
@@ -711,5 +711,67 @@ CREATE TABLE IF NOT EXISTS iiq_schema.async_profile_analytics
 
     CONSTRAINT unique_async_profile_analytics UNIQUE (iiq_id)
 );
-
 ALTER TABLE iiq_schema.async_profile_analytics OWNER TO iiq;
+
+
+CREATE TABLE IF NOT EXISTS iiq_schema.async_contents_fetch_data
```

Comment:
async_contents_fetch_request.id or iiq_id should be present in this table, which signifies the request for which this content was populated